src/core_plugins/metric_vis/public/metric_vis_params.html

```diff
@@ -119,16 +150,23 @@
         }"
         class="fa fa-caret-right kuiSideBarCollapsibleTitle__caret"
       ></span>
-        <span class="kuiSideBarCollapsibleTitle__text">
-        Color Options
+        <span
+          class="kuiSideBarCollapsibleTitle__text"
+          i18n-id="metricVis.public.metricVisParams.color.colorOptionsTitle"
+          i18n-default-message="Color Options"
+        >
       </span>
       </div>
     </div>
     <div id="metricOptionsColors" ng-if="showColorOptions" class="kuiSideBarCollapsibleSection">
       <div class="kuiSideBarSection">
         <div class="kuiSideBarFormRow" ng-show="editorState.params.metric.colorsRange.length > 1">
-          <label class="kuiSideBarFormRow__label" for="metricColorMode">
-            Use Color For
+          <label
+            class="kuiSideBarFormRow__label"
+            for="metricColorMode"
+            i18n-id="metricVis.public.metricVisParams.color.useForLabel"
+            i18n-default-message="Use Color For"
+          >
           </label>
           <div class="kuiSideBarFormRow__control">
             <select
```

Comment:
Don't we want to translate the color mode values? Since it has values like `None`, `Labels`, `Background`? cc @alexfrancoeur